src/core/monitors/command.js

```diff
@@ -10,6 +10,10 @@ class CommandHandler extends Monitor {
   constructor(...args) {
     super(...args);
     this.ignoreEdits = !this.client.options.commandEditing;
+    this.prefixReminder = null;
+    this.client.on("ready", () => {
```

Comment:
This is really not needed here. If you want to listen for an emit to use, you should use an actual event file. Either way, the old method worked fine.

My thought here was to assign a mention prefix to the main ayame client and setting the prefix once the client was ready. Then you would just check to see if the prefix is equal to the mentionPrefix here.

Reply:
I'd keep the normal comparison I did.

Reply:
Doesn’t work well, regex is easier. The original method returned false.